TrkHitReco/data/TrainBkgDiagSTHplane.dat

```diff
@@ -0,0 +1,16 @@
+tensor_dense3kernel0 18
```

Comment:
And name this TrainBkgDiagPlane?  Then the existing TrkBiagDiag can be called TrkBkgDiagPanel and everything is explicit.